src/rtScript.h

```diff
@@ -74,6 +79,9 @@ class rtIScript
   virtual rtError pump() = 0;
 
   virtual rtError collectGarbage() = 0;
+#ifdef ENABLE_DEBUG_MODE
```

Comment:
Rather than defining methods or not defining methods depending on build defines, let's instead always define these methods, and let's have the methods themselves log a message if the ability is disabled and take no action.  Please follow the same pattern for all method declarations and definitions.